gprofiler/profilers/java.py

```diff
@@ -698,8 +707,11 @@ def __init__(
         self._metadata = JavaMetadata(self._stop_event)
         self._insert_dso_name = insert_dso_name
 
-    def _init_ap_mode(self, ap_mode: str) -> None:
-        if ap_mode == "auto":
+    def _init_ap_mode(self, profiling_mode: str, ap_mode: str) -> None:
+        if profiling_mode == "allocation":
+            ap_mode = "alloc"
```

Comment:
Sooo - I ran this mode and got `_[i]` frames, what are these? Maybe okay to leave handling of them for later (e.g categorizing in the backend)

Reply:
According to async-profiler `_[i]` frames are normal frame names, `_[k]` is used for allocation outside of TLAB

Reply:
I think for basic allocation profiling (this PR), we can keep it simple without parsing or adding special handling of this case

Reply:
Okay, although, it will look weird in the UI at first. but kk